src/util/temp_dir.rs

```diff
@@ -14,7 +14,7 @@ pub struct TempDir {
 }
 
 impl TempDir {
-	/// Create a new temporary directory in the system tempdir.
+	/// Create a new temporary directory in the system tempdir. (for unix this is /tmp)
```

Comment:
```suggestion
	/// Create a new temporary directory in the system tempdir.
```

The exact rules are more complicated, so I would rather not document them at all.